src/SFA.DAS.AANHub.Application.UnitTests/Members/Queries/GetMembersQueryHandlerTests.cs

```diff
@@ -177,6 +178,41 @@ public async Task Handle_UserType_CheckUserTypeExpected(List<MemberUserType> use
             It.Is<GetMembersOptions>(c => c.UserType == userType), cancellationToken), Times.Once);
     }
 
+    [Test, RecursiveMoqAutoData]
+    public async Task Handle_UserType_ShouldNotReturnMemberWithAnyUserTypeExceptEmployerAndApprentice(
```

Comment:
Your change is in the repository so having this test on the handler makes no sense